stix2elevator/convert_cybox.py

```diff
@@ -233,21 +251,22 @@ def convert_hashes(hashes):
 
 
 def convert_windows_executable_file(f):
-    dict = {}
+    w_ex_dict = {}
     if f.headers:
         file_header = f.headers.file_header
         if file_header:
             for prop_tuple in _PE_FILE_HEADER_PROPERTY_MAP:
                 prop_name1x = prop_tuple[0]
                 prop_name2x = prop_tuple[1]
-                dict[prop_name2x] = getattr(file_header, prop_name1x)
+                if hasattr(file_header, prop_name1x) and getattr(file_header, prop_name1x):
```

Comment:
```suggestion
                if hasattr(file_header, prop_name1x):
```

Reply:
The way the stix1x parse works - there will almost always be a property - but it will often be None.  So doing both is just safer.

Reply:
Would it be better to use this then?
```suggestion
                if getattr(file_header, prop_name1x, None):
```
If the property doesn't exist it'll use the third parameter there as the default.

Reply:
sure, I will do these, but there are probably other places where that tweak would be useful - but let's do that after the release.